src/webui/webapplication.cpp

```diff
@@ -95,33 +95,32 @@ namespace
         return ret;
     }
 
-    void translateDocument(QString &data)
+    void translateDocument(const QString &locale, QString &data)
     {
         const QRegExp regex("QBT_TR\\((([^\\)]|\\)(?!QBT_TR))+)\\)QBT_TR(\\[CONTEXT=([a-zA-Z_][a-zA-Z0-9_]*)\\])");
         const QRegExp mnemonic("\\(?&([a-zA-Z]?\\))?");
-        int i = 0;
-        bool found = true;
 
-        const QString locale = Preferences::instance()->getLocale();
-        bool isTranslationNeeded = !locale.startsWith("en") || locale.startsWith("en_AU") || locale.startsWith("en_GB");
+        const bool isTranslationNeeded = !locale.startsWith("en")
+            || locale.startsWith("en_AU") || locale.startsWith("en_GB");
```

Comment:
why `||`? both strings also start with "en", so them are handled by first condition

Reply:
sorry, my bad. I didn't noticed `!` in first condition...